sphinx/themes/basic/static/searchtools.js

```diff
@@ -90,10 +99,10 @@ const _displayItem = (item, searchTerms, highlightTerms) => {
   let linkEl = listItem.appendChild(document.createElement("a"));
   linkEl.href = linkUrl + anchor;
   linkEl.dataset.score = score;
-  linkEl.innerHTML = title;
+  linkEl.innerHTML = _escapeHTML(title);
   if (descr) {
     listItem.appendChild(document.createElement("span")).innerHTML =
-      " (" + descr + ")";
+      " (" + _escapeHTML(descr) + ")";
```

Comment:
```suggestion
      _escapeHTML(" (" + descr + ")");
```

Nitpicky, and maybe redundant: however I think this would make it easier to communicate that we should HTML-escape all content before assigning it to `innerHTML` (e.g. for anyone else copying/reading the code later).

Reply:
Yep this is better!